api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -822,12 +968,75 @@ public void testGetProjectCountByStatus() {
     assertThat(statusToCount).hasSize(2);
   }
 
-  private DbBillingProjectBufferEntry makeSimpleEntry(BufferEntryStatus bufferEntryStatus) {
-    DbBillingProjectBufferEntry entry = new DbBillingProjectBufferEntry();
-    entry.setStatusEnum(bufferEntryStatus, () -> Timestamp.from(CLOCK.instant()));
+  @Test
+  public void testFindEntriesWithExpiredGracePeriod() {
```

Comment:
I felt this existing method needed a test.  It's not something new for multi-tier.